Auth0/Management/ObjectiveC/_ObjectiveManagementAPI.swift

```diff
@@ -68,6 +68,24 @@ public class _ObjectiveManagementAPI: NSObject {
                 }
         }
     }
+    
+    @objc(unlinkUserWithIdentifier:withProvider:fromUserId:callback:)
```

Comment:
@sebacancinos let's rename the objc method to `unlinkIdentityWithIdentifier:provider:fromUserId:callback`